Core/src/main/java/com/plotsquared/core/util/MainUtil.java

```diff
@@ -44,6 +44,53 @@
     private static final Logger logger =
         LoggerFactory.getLogger("P2/" + MainUtil.class.getSimpleName());
 
+    /**
```

Comment:
This was done to make it compile as @dordsor21 left the project un-compilable. This will be removed before merge.